src/org/labkey/test/components/ui/grids/FieldReferenceManager.java

```diff
@@ -133,34 +154,12 @@ public WebElement getElement()
 
         public FieldKey getFieldKey()
         {
-            if (_fieldKey.getValue() == null)
-            {
-                String path = _element.getDomAttribute("data-fieldkey");
-                if (path == null)
-                {
-                    // Some grids don't have a field key, but have a similar value in the ID attribute
-                    path = _element.getDomAttribute("id");
-                }
-
-                if (path != null)
-                {
-                    _fieldKey.setValue(FieldKey.fromFieldKey(path));
-                }
-                else
-                {
-                    _fieldKey.setValue(FieldKey.EMPTY);
-                }
-            }
-            return _fieldKey.getValue();
+            return _fieldKey.get();
```

Comment:
Why is this variable preceded with an _ but and earlier change removed the underscore from a variable _fieldReferences to fieldReferences

Reply:
Good question. I'll switch these to use `_`.